aws-iam-role/README.md

```diff
@@ -42,7 +42,7 @@ module iam-role {
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
-| attached\_policies\_arns | List of policies ARNs to be attached to the IAM role. | `list(string)` | `[]` | no |
+| attached\_policies\_names\_arns | Map of policy names to the respective ARNs to be attached to the IAM role. | `map` | `{}` | no |
```

Comment:
overall lgtm, (non-blocking) this requires ppl to input the unnecessary names though, I wonder if we can make this more straightforward by filling out the map itself in the module, so ppl still provides the list in the module and the module converts the list into a map. but overall, current version lgtm!

Reply:
Yeah that's a good idea, thanks! I'll do that